compiler/rustc_middle/src/ty/context.rs

```diff
@@ -3373,19 +3370,6 @@ impl<'tcx> TyCtxt<'tcx> {
         self.opt_rpitit_info(def_id).is_some()
     }
 
-    /// Named module children from all kinds of items, including imports.
-    /// In addition to regular items this list also includes struct and variant constructors, and
-    /// items inside `extern {}` blocks because all of them introduce names into parent module.
-    ///
-    /// Module here is understood in name resolution sense - it can be a `mod` item,
-    /// or a crate root, or an enum, or a trait.
-    ///
-    /// This is not a query, making it a query causes perf regressions
-    /// (probably due to hashing spans in `ModChild`ren).
```

Comment:
Now I remember, I tried to querify this before and it led to similar regressions.
Perhaps `eval_always` will help a bit.

Reply:
One alternative would be to hash `resolutions`, as there is no more unstable information inside it.